packages/utilities/src/EventGroup.ts

```diff
@@ -307,10 +307,10 @@ export class EventGroup {
     return EventGroup.raise(this._parent, eventName, eventArgs, bubbleEvent);
   }
 
+  //see https://github.com/alangpierce/sucrase/issues/545
   /** Declare an event as being supported by this instance of EventGroup. */
-  public declare(event: string | string[]): void {
+  public ['declare'](event: string | string[]): void {
```

Comment:
I'd recommend adding a short inline explanation for the use of quotes, in addition to the link. Otherwise it's not clear what the link is referring to.